diff --git a/project_config/lmic_project_config.h b/project_config/lmic_project_config.h
index 732e6e6..de84e61 100644
--- a/project_config/lmic_project_config.h
+++ b/project_config/lmic_project_config.h
@@ -1,10 +1,13 @@
 // project-specific definitions
-//#define CFG_eu868 1
-#define CFG_us915 1
+#define CFG_eu868 1
+//#define CFG_us915 1
 //#define CFG_au915 1
 //#define CFG_as923 1
 // #define LMIC_COUNTRY_CODE LMIC_COUNTRY_CODE_JP	/* for as923-JP */
 //#define CFG_kr920 1
 //#define CFG_in866 1
 #define CFG_sx1276_radio 1
 //#define LMIC_USE_INTERRUPTS
+#define LMIC_LORAWAN_SPEC_VERSION   LMIC_LORAWAN_SPEC_VERSION_1_0_2
+#define DISABLE_BEACONS
+#define DISABLE_PING
\ No newline at end of file
